### PR TITLE
Configure Travis CI

### PR DESCRIPTION
**Pull request by cabarnes**
Add configuration file for Travis CI

### CODE REVIEW

**Comment by btimby** [comment]
Holy crap, I don't look forward to updating `eslint` and re-linting `ftpserver`.

File: lib/rest/basic.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const { Client } = require('./client');
+
+class BasicClient extends Client {
+  constructor(opts) {
+    // eslint-disable-next-line prefer-object-spread
```
Comment (btimby) [inline lib/rest/basic.js]: I don't know how to fix these. When I follow the recommendations, it fails.

Comment (cabarnes) [reply lib/rest/basic.js]: I'm not sure why the recommendations were written the way they were. I removed the `assign` calls and used object spread and it seems to be happy.

Comment (btimby) [reply lib/rest/basic.js]: It's because I am a dummy. I did `Object.assign(...foo)`. IOW, I used both.

**Pull request merged**